### PR TITLE
Correction of quote for French + space before colon

### PR DESCRIPTION
**Pull request by Wadeck**
- correction of a `\'` in french
- add a space before `:`, in english only one space after, in french one before and one after

As the colon moved from HTML to translation (properties file), I let original translator validates the addition of colon in their language.
- [ ] @alshopov for bulgarian translation (according to [Microsoft Bulgarian Style Guide](https://www.google.ch/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&cad=rja&uact=8&ved=0ahUKEwih5NrfivrXAhWFPxoKHcMlDkkQFggnMAA&url=http%3A%2F%2Fdownload.microsoft.com%2Fdownload%2Fa%2F2%2F9%2Fa29902cc-41c0-4ceb-a6ed-ba16bc511e15%2Fbul-bgr-styleguide.pdf&usg=AOvVaw2TnsdLGRd8mrTc3waBIlJj) it's like in english)
- [ ] @tomas for lithuanian translation (nothing special mentioned in [Microsoft Lithunian Style Guide](https://www.google.ch/url?sa=t&rct=j&q=&esrc=s&source=web&cd=2&cad=rja&uact=8&ved=0ahUKEwjM-IrEi_rXAhWKfxoKHZKwALYQFggvMAE&url=http%3A%2F%2Fdownload.microsoft.com%2Fdownload%2Fd%2F9%2Fb%2Fd9b3b746-78ec-4049-bc43-c0f744199f71%2Flit-ltu-StyleGuide.pdf&usg=AOvVaw1kRbldbyOy83vobZySPtYX), so like english ?)
- [x] @damianszczepanik for polish translation (according to [Microsoft Polish Style Guide](https://www.google.ch/url?sa=t&rct=j&q=&esrc=s&source=web&cd=2&cad=rja&uact=8&ved=0ahUKEwi4uIX_i_rXAhXGMBoKHSzZAC0QFggtMAE&url=http%3A%2F%2Fdownload.microsoft.com%2Fdownload%2F3%2Fe%2F6%2F3e688569-1e3f-48a4-a33c-07e6daaa57b4%2Fpol-pol-StyleGuide.pdf&usg=AOvVaw30jwInULP3yzHI7BtbgpMb) it's like in english)

@reviewbybees

### CODE REVIEW

File: core/src/main/resources/hudson/model/View/newJob_fr.properties [hunk 1]
```diff
@@ -26,7 +26,7 @@ CopyExisting=Copier un {0} existant
 ItemName.help=Champ obligatoire
 ItemName.label=Saisissez un nom
 ItemName.validation.required=Ce champ ne peut pas \u00eatre vide. Veuillez saisir un nom valide et appuyer sur OK.
-ItemType.validation.required=Veuillez choisir un type d\'Item
+ItemType.validation.required=Veuillez choisir un type d\u2019Item
```
Comment (alecharp) [inline core/src/main/resources/hudson/model/View/newJob_fr.properties]: maybe we can translate "Item" as well

**Review by alecharp — CHANGES_REQUESTED** [review]

**Review by alecharp — APPROVED** [review]
LGTM :bee:

**Comment by Wadeck** [comment]
@reviewbybees done

**Comment by oleg-nenashev** [comment]
Hmm, it impacts other translations as well. 
Let's wait for responses from CCed translation contributors

**Comment by damianszczepanik** [comment]
As I see the colon has been moved from template to translations so it should not have impact for translations that were updated (hopefully all). So for Polish translation - fix is fine.

**Comment by Wadeck** [comment]
@damianszczepanik yeah all the translations where I can read the characters that have a colon were updated (and CC in the description)
Thank you for the review 👍

**Review by damianszczepanik — APPROVED** [review]

**Pull request merged**